content/docs/code-splitting.md

```diff
@@ -40,86 +45,90 @@ function add(a, b) {
 console.log(add(16, 26)); // 42
 ```
 
-> Note:
+> Зауважте:
 >
-> Your bundles will end up looking a lot different than this.
+> Наприкінці ваші бандли будуть
+> значно відрізнятися від наведених прикладів.
```

Comment:
@klymenkoo should this be 1 line?

Reply:
@ApolllonDev , fixed to one line, you are right